packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,161 @@
+| `tfe_vault_disable_mlock`                    | no       | `"false"`                                    | Disable mlock for internal Vault.                                                                                                                                                                                                                                                   |
+| `tfe_resource_cpu`                           | no       | `750`                                        | CPU in MHz for TFE container.                                                                                                                                                                                                                                                       |
+| `tfe_resource_memory`                        | no       | `1024`                                       | Memory in MB for TFE container.                                                                                                                                                                                                                                                     |
+| `tfe_image`                                  | no       | `"hashicorp/terraform-enterprise:v202401-2"` | TFE image and tag to download and run.                                                                                                                                                                                                                                              |
+| `tfe_image_registry_username`                | no       | `"terraform"`                                | The user name for the registry where the TFE image is hosted.                                                                                                                                                                                                                       |
+| `tfe_image_server_address`                   | no       | `"images.releases.hashicorp.com"`            | The server address of the registry where TFE image is hosted.                                                                                                                                                                                                                       |
+| `tfe_run_pipeline_nomad_tls_config_insecure` | no       | `"false"`                                    | mTLS between Nomad and TFE when set to false.                                                                                                                                                                                                                                       |
```

Comment:
```suggestion
| `tfe_run_pipeline_nomad_tls_config_insecure` | no       | `false`                                    | mTLS between Nomad and TFE when set to false.                                                                                                                                                                                                                                       |
```

Reply:
`variable "tfe_run_pipeline_nomad_tls_config_insecure" {
  description = "The tls config settings for communication between Nomad and TFE"
  type        = string
  default     = "false"
}`

I think it expects a string.